llvm/lib/Target/SBF/BTFDebug.h

```diff
@@ -34,10 +35,6 @@ class MCInst;
 class MCStreamer;
 class MCSymbol;
 
-namespace BTFX {
-
-class BTFDebug;
-
```

Comment:
As above, we need to restore this fragment for BTFX namespace (with BTFDebug sunk as shown).